pkg/api/validation/validation_test.go

```diff
@@ -3179,38 +3179,49 @@ func TestValidatePod(t *testing.T) {
 				//			"namespaces":["ns"],
 				//			"topologyKey": "zone"
 				//		}]
-				Annotations: map[string]string{
-					api.AffinityAnnotationKey: `
-					{"podAffinity": {
-						"requiredDuringSchedulingIgnoredDuringExecution": [{
-							"labelSelector": {
-								"matchExpressions": [{
-									"key": "key2",
-									"operator": "In",
-									"values": ["value1", "value2"]
-								}]
+			},
+			Spec: api.PodSpec{
+				Containers:    []api.Container{{Name: "ctr", Image: "image", ImagePullPolicy: "IfNotPresent"}},
```

Comment:
So I'm guessing this section is a copy<>paste of validPodSpec?  b/c it's repeated.